app/controllers/UsersController.scala

```diff
@@ -168,13 +168,18 @@ class UsersController @Inject()(messagesApi: MessagesApi,
     )
   }
 
-  def login: Action[AnyContent] = action { implicit request =>
-    Ok(views.html.users.login(loginForm))
+  def login: Action[AnyContent] = action.async { implicit request =>
+    val username = configuration.get[String]("session.username")
```

Comment:
Can we make "username" as a class level variable instead? It is not going to change often.